src/gc/handletable.cpp

```diff
@@ -21,6 +21,8 @@
 #include "objecthandle.h"
 #include "handletablepriv.h"
 
+#define COUNTER_ONLY(x)
```

Comment:
Just change the single place where this is used to `#ifdef ENABLE_PERF_COUNTERS`

Reply:
And delete `src\gc\env\gcenv.base.h(501): #define COUNTER_ONLY(x)` to make the `COUNTER_ONLY` macro to disappear completely. We are just going to have `ENABLE_PERF_COUNTERS` in the gc.